docs/src/convergence.md

```diff
@@ -0,0 +1,119 @@
+# [Experimental convergence order of MPRK schemes](@id convergence_mprk)
+
+## Second order MPRK schemes
+
+```@example eoc
+using PositiveIntegrators
+
+# choose schemes
+algs = [MPRK22(0.5)
+        MPRK22(2.0 / 3.0)
+        MPRK22(1.0)
+        SSPMPRK22(0.5, 1.0)]
+
+names = ["MPRK22(0.5)"
+         "MPRK22(2.0/3.0)"
+         "MPRK22(1.0)"
+         "SSPMPRK22(0.5, 1.0)"]
+
+prob = prob_pds_linmod
+
+#nothing #hide
+#```
+#```@example eoc
+using DiffEqDevTools
+
+dts = 0.5 .^ (5:10)
+err = Vector{Vector{Float64}}(undef, length(algs))
+eoc = Vector{Vector{Float64}}(undef, length(algs))
+
+#compute errors and experimental order of convergence
+for i in eachindex(algs)
+    sim = test_convergence(dts, prob, algs[i])
+    err[i] = sim.errors[:l∞]
+    eoc[i] = -log2.(err[i][2:end] ./ err[i][1:(end - 1)])
+end
+#```
+#
+#```@example eoc
+using PrettyTables
+
+# collect data and create headers
```

Comment:
Coding this almost drove me crazy! 

Originally I used something like `data = dts` outside the loop and `data = [data err[i] [NaN; eoc[i]]]` within the loop. This was perfectly fine on my local machine, but caused an error like "data not defined" in the build process.

@ranocha: Any idea what the problem was/is?

Reply:
Did you built the docs locally or copy and paste the code to the REPL?

Reply:
Oh. I didn't build the docs locally, I copied & pasted everything from my vscode repl to the markdown files. But I ensured to use the docs environment.

Reply:
Then it might be an issue of soft global scope, see https://docs.julialang.org/en/v1/manual/variables-and-scoping/#on-soft-scope

Reply:
You may need to add a keyword `global`